Habitica/src/main/java/com/habitrpg/android/habitica/ui/fragments/BaseFragment.java

```diff
@@ -133,5 +108,32 @@ public String getDisplayedClassName() {
         return this.getClass().getSimpleName();
     }
 
+    private class TutorialStepTransactionListener implements TransactionListener<TutorialStep> {
```

Comment:
I think it's the other way around? non-static inner classes have a reference to the outer class, but static ones don't.